README.md

```diff
@@ -11,13 +11,13 @@ This package is available for several languages/platforms:
 To use from JavaScript or TypeScript in Node.js, install using either `npm`:
 
 ```bash
-npm install @vavsab/pulumi-zitadel
+npm install @pulumiverse/pulumi-zitadel
```

Comment:
The correct NPM package name is `@pulumiverse/zitadel`.

Reply:
Fixed